src/go/rpk/pkg/cli/cluster/health.go

```diff
@@ -72,12 +72,13 @@ following conditions are met:
 func printHealthOverview(hov *adminapi.ClusterHealthOverview) {
 	types.Sort(hov)
 	out.Section("CLUSTER HEALTH OVERVIEW")
-	overviewFormat := `Healthy:               %v
+	overviewFormat := `Healthy:                     %v
```

Comment:
Sneaking this change in: I adjusted the `Healthy:` line because it wasn't lined up with the remaining fields below after the alignment changed (probably because it's not on a line of its own in the source so it was missed).